src/application/utils.ts

```diff
@@ -1,3 +1,11 @@
 export function h2b(hex: string): Buffer {
   return Buffer.from(hex, 'hex');
 }
+
+export function b2hReversed(buffer: Buffer): string {
+  return Buffer.from(buffer).reverse().toString('hex');
```

Comment:
Why
`Buffer.from(buffer).reverse().toString('hex');`
instead of
`buffer.reverse().toString('hex');`
?

Reply:
`reverse()` function mutates the original buffer/array. It led to lot of errors in the past. `Buffer.from` is a trick to "copy" the buffer and create a new one (thus reverse is not applied on `buffer`)